src/org/thoughtcrime/securesms/ConversationListFragment.java

```diff
@@ -491,31 +611,32 @@ public void onSwiped(RecyclerView.ViewHolder viewHolder, int direction) {
 
       if (archive) {
         new SnackbarAsyncTask<Long>(getView(),
-                                    getResources().getQuantityString(R.plurals.ConversationListFragment_moved_conversations_to_inbox, 1, 1),
-                                    getString(R.string.ConversationListFragment_undo),
-                                    getResources().getColor(R.color.amber_500),
-                                    Snackbar.LENGTH_LONG, false)
+                getResources().getString(R.string.ConversationListFragment_deleting),
+                "",
+                getResources().getColor(R.color.amber_500),
+                Snackbar.LENGTH_LONG, false)
+
         {
           @Override
           protected void executeAction(@Nullable Long parameter) {
-            DatabaseFactory.getThreadDatabase(getActivity()).unarchiveConversation(threadId);
+            DatabaseFactory.getThreadDatabase(getActivity()).deleteConversation(threadId);
           }
 
           @Override
           protected void reverseAction(@Nullable Long parameter) {
-            DatabaseFactory.getThreadDatabase(getActivity()).archiveConversation(threadId);
+            //unused
```

Comment:
Can we remove this or does it need to be overridden?

Reply:
Why do we not have a reverse option? Maybe we could put the conversation in temporary storage?

Reply:
That could be a good idea for a future story. Currently any kind of delete is irreversible in Signal, so we went with that for this story.